api/models.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import unicode_literals
 
 import json
+from datetime import datetime
```

Comment:
Ah okay, nice to know! I replaced `datetime.datetime` with `django.utils.timezone`.